source/MaterialXCore/Interface.cpp

```diff
@@ -337,7 +337,7 @@ bool Input::validate(string* message) const
     if (parent->isA<Node>())
     {
         bool hasValueBinding = hasValue();
-        bool hasConnection = hasNodeName() || hasNodeGraphString() || hasOutputString() || hasInterfaceName();
+        bool hasConnection = hasNodeName() || hasNodeGraphString() || hasOutputString() || hasInterfaceName() || hasDefaultGeomPropString();
```

Comment:
I'm not sure this change makes sense, since node inputs don't support `defaultgeomprop` strings, so the presence of one wouldn't constitute a meaningful value or connection for the node input.

Is there a case that I'm missing here, where `defaultgeomprop` might be meaningful on a node input, as opposed to its more traditional usage on a nodedef input?

Reply:
This is currently producing erroneous warning, so I guess an additional check for if it's a `nodedef` input should be added.
Currently tests fail since we skip on validation warnings.

Reply:
I'm not seeing any warnings or errors when I run the render test suite for MaterialX, and I'd imagine it would represent a real error if one appeared in a future build, so I'd recommend that we fix the error rather than suppressing it here.